legend-sdlc-server/src/main/java/org/finos/legend/sdlc/server/gitlab/api/GitLabProjectApi.java

```diff
@@ -456,6 +417,8 @@ public ImportReport importProject(String id, ProjectType type, String groupId, S
             }
             catch (Exception e)
             {
+                // Try to delete the branch in case of exception
+                deleteWorkspace(projectId, repositoryApi, workspaceId);
```

Comment:
We should also try to close the MR at this point, probably before trying to delete the branch. As with the attempt to delete the branch, failure to close the MR should be logged but should not throw an exception.

Reply:
If it's automatically closed by deleting the branch, then we don't need to explicitly close it.